setup/readme.md

```diff
@@ -1,11 +1,14 @@
-systemd service file for cloudfuse
+# systemd service file for cloudfuse
 
-# Step to install this to systemd
-1. If you are existing blobfuse user follow the MIGRATION.md file on how to convert blobfuse config and cli parameters to cloudfuse compliant config
-2. Prepare the config file. Name of the container should be specified by `azstorage:container` in config file or AZURE_STORAGE_ACCOUNT_CONTAINER environment variable. 
-3. Download cloudfuse.service and put it into /etc/systemd/system
-4. Edit the file, changing environment values in Service block.
-5. Run command to reload service config files: `systemctl daemon-reload`
-6. Start service: `systemctl start cloudfuse.service`
-7. Make the service starting with system: `systemctl enable cloudfuse.service`
-8. Please Note that the example has the User AzureUser, please create a user called AzureUser or replace this with an existing user.
+## Steps to install Cloudfuse to systemd
+
+1. Prepare the configuration file `config.yaml` that cloudfuse will use to start your mount. Follow the general instructions for creating a config.yaml file from the Readme.
+2. In the cloudfuse.service file, edit all of the fields under the `Service` section and replace them for your system. Please note that the example has the User CloudfuseUser, please create a user called CloudfuseUser or replace this with an existing user.
```

Comment:
I think we need to add the cloudfuse.service file to our deployment pipeline so it makes it into our Linux builds.
Relatedly, I'm not sure: should we push it to /etc/systemd ourselves? That might be the common practice.

Reply:
I agree. After doing some reading it looks like we could write to /usr/lib/systemd/ which seems to be where these files should be placed for installed software https://unix.stackexchange.com/questions/206315/whats-the-difference-between-usr-lib-systemd-system-and-etc-systemd-system. I will add this to the backlog.